apps/platform-app/src/app/[locale]/(profile)/_workflows/profile-evaluation.workflow.ts

```diff
@@ -0,0 +1,152 @@
+'use server'
+import { AIMessage } from '@langchain/core/messages'
+import { ChatGroq } from '@langchain/groq'
+import { END, START, StateGraph } from '@langchain/langgraph'
+import { ToolNode } from '@langchain/langgraph/prebuilt'
+
+import {
+  sendProfileApprovedEmail,
+  sendProfileRejectedEmail,
+} from '@/backend/mailing/mailing.service'
+import { updateProfileById } from '@/backend/profile/profile.service'
+import { getUserById } from '@/backend/user/user.service'
+import { type UserWithProfileAndGH } from '@/backend/user/user.types'
+import { getContextVariable, setContextVariable } from '@langchain/core/context'
+import { tool } from '@langchain/core/tools'
+import { Annotation, type LangGraphRunnableConfig } from '@langchain/langgraph'
+import { z } from 'zod'
+
+const StateAnnotation = Annotation.Root({
+  user: Annotation<UserWithProfileAndGH>(),
```

Comment:
**nit:** we would rather need `profile` not `user`. User will be included in profile anyway